### PR TITLE
Feature/sbachmei/mic 3376 ldlc medication effects

### PR DESCRIPTION
**Pull request by stevebachmeier**
### Description
- *Category*: Implementation
- *JIRA issue*: [MIC-3376](https://jira.ihme.washington.edu/browse/MIC-3376)
- *Research reference*: [Treatment Effects](https://vivarium-research.readthedocs.io/en/latest/concept_models/vivarium_us_cvd/concept_model.html#treatment-effects)

Currently we've provided simulant exposure to ldlc medication but that
doesn't actually affect anything. This PR adds the ldlc pipeline
modifier and adds the effect of the treatment.

Note that unlike sbp medication - which is an additive effect - this
is a multiplicative effect.

### Verification and Testing
Spot-checked.

### CODE REVIEW

File: src/vivarium_nih_us_cvd/components/risks.py [hunk 1]
```diff
@@ -11,26 +11,15 @@
 
 
 class Risk(Risk_):
```
Comment (rmudambi) [inline src/vivarium_nih_us_cvd/components/risks.py]: Give this class a more descriptive name. What makes it different from the Risk class you've inherited from. That will help someone reading the code understand why it was necessary to make these changes. It'll also help you know quickly whether or not you should use this risk component for BMI and FPG or not.

File: src/vivarium_nih_us_cvd/components/treatment.py [hunk 2]
```diff
@@ -156,17 +161,61 @@ def adjust_target(index: pd.Index, target: pd.Series) -> pd.Series:
 
         return adjust_target
 
-    def _register_target_modifier(self, builder: Builder) -> None:
+    def _get_ldlc_medication_effects(self, builder: Builder) -> Dict[str, float]:
+        """Format the ldlc medication effects data"""
+        effects = builder.data.load("risk_factor.high_ldl_choldesterol.medication_effect")
```
Comment (rmudambi) [inline src/vivarium_nih_us_cvd/components/treatment.py]: typo "choldesterol" in this key

File: src/vivarium_nih_us_cvd/components/treatment.py [hunk 3]
```diff
@@ -205,6 +254,33 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
             (mask_sbp_adherent) & (mask_sbp_two_drugs), data_values.COLUMNS.SBP_MULTIPLIER
         ] = data_values.SBP_MULTIPLIER.TWO_DRUGS
 
+        pop[data_values.COLUMNS.LDLC_MULTIPLIER] = 1
+        mask_ldlc_adherent = (
+            pop[data_values.COLUMNS.LDLC_MEDICATION_ADHERENCE]
+            == data_values.MEDICATION_ADHERENCE_TYPE.ADHERENT
+        )
+        mask_ldlc_low = (
```
Comment (rmudambi) [inline src/vivarium_nih_us_cvd/components/treatment.py:262]: Consider including adherence in these masks so you don't have to use it later below. It's fine either way though.

File: src/vivarium_nih_us_cvd/constants/data_keys.py [hunk 4]
```diff
@@ -171,6 +171,9 @@ class __HighLDLCholesterol(NamedTuple):
     RELATIVE_RISK_SCALAR: TargetString = TargetString(
         "risk_factor.high_ldl_cholesterol.relative_risk_scalar"
     )
+    MEDICATION_EFFECT: TargetString = TargetString(
+        "risk_factor.high_ldl_choldesterol.medication_effect"
```
Comment (rmudambi) [inline src/vivarium_nih_us_cvd/constants/data_keys.py]: Typo here again.

File: src/vivarium_nih_us_cvd/model_specifications/nih_us_cvd.yaml [hunk 5]
```diff
@@ -3,7 +3,7 @@ components:
         population:
             - BasePopulation()
             - Mortality()
-        metrics:
+        # metrics:
```
Comment (rmudambi) [inline src/vivarium_nih_us_cvd/model_specifications/nih_us_cvd.yaml]: Why is this commented out?

File: src/vivarium_nih_us_cvd/utilities.py [hunk 6]
```diff
@@ -9,7 +9,7 @@
 from vivarium.framework.randomness import get_hash
 from vivarium_public_health.risks.data_transformations import pivot_categorical
 
-from vivarium_nih_us_cvd.constants import data_values, metadata
+from vivarium_nih_us_cvd.constants import metadata
```
Comment (rmudambi) [inline src/vivarium_nih_us_cvd/utilities.py:12]: Is this actually used?

**Review by rmudambi — APPROVED** [review]
Everything looks good except the class name

File: src/vivarium_nih_us_cvd/utilities.py (hunk 6 above)
Comment (stevebachmeier) [reply src/vivarium_nih_us_cvd/utilities.py:12]: Yes, line 24 (len_longest_location())

**Pull request merged**